assets/js/storage.js

```diff
@@ -1,6 +1,8 @@
 angular
 .module('storage', [])
 .factory('$storage', function() {
+  var likedVideos = getObj('likes');
```

Comment:
This was the exact issue I was running into: https://docs.angularjs.org/error/$rootScope/infdig